parlai/scripts/convo_render.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import sys
+import json
+import random
+import argparse
+import tempfile
+
+# Constant Denoting the End of a Conversation
+END_OF_CONVO = "EOC"
+
+
+def gen_ul_style():
+    """ Generate general ul tag style"""
+    return (
+        "\t\tul{\n\t\t\tlist-style: none;\n\t\t\tmargin: 0;\n\t\t\tpadding: 0;\n\t\t}\n"
+    )
+
+
+def gen_ul_li_style():
+    """ Generate style for the li items for the conversations"""
+    return "\t\tul li{\n\t\t\tdisplay:inline-block;\n\t\t\tclear: both;\n\t\t\tpadding: 20px;\n\t\t\tborder-radius: 30px;\n\t\t\tmargin-bottom: 2px;\n\t\t\tfont-family: Helvetica, Arial, sans-serif;\n\t\t}\n"
+
+
+def gen_breaker_style():
+    """ Generate style for the break li item between conversations"""
+    return "\t\t.breaker{\n\t\t\tcolor: #bec3c9;\n\t\t\tdisplay: block;\n\t\t\theight: 20px;\n\t\t\tmargin: 20px 20px 20px 20px;\n\t\t\ttext-align: center;\n\t\t\ttext-transform: uppercase;\n\t\t}\n"
+
+
+def gen_speaker_styles(other_speaker, human_speaker):
+    """
+    Generate style string for the speakers
+    :param other_speaker: The title of the model (grey boxes)
+    :param human_speaker: Human speaker in the dialogs (blue boxes)
+
+    :return: Style string for the speakers
+    """
+    other_style = (
+        "\t\t."
+        + other_speaker
+        + "{\n\t\t\tbackground: #eee;\n\t\t\tfloat: left;\n\t\t}\n"
+    )
+    human_style = (
+        "\t\t."
+        + human_speaker
+        + "{\n\t\t\tbackground: #0084ff;\n\t\t\tcolor: #fff;\n\t\t\tfloat: right;\n\t\t}\n"
+    )
+
+    return other_style + human_style
+
+
+def gen_style_str(height, width, other_speaker, human_speaker):
+    """
+    Generate the style section string of the HTML
+    :param height: Height of the HTML page
+    :param width: Width of the HTML page
+    :param other_speaker: The title of the model (grey boxes)
+    :param human_speaker: Human speaker in the dialogs (blue boxes)
+
+    :return: The <style> section string
+    """
+    ul_style = gen_ul_style()
+    ul_li_style = gen_ul_li_style()
+    speaker_styles = gen_speaker_styles(other_speaker, human_speaker)
+    breaker_style = gen_breaker_style()
+    style_str = (
+        '\t<style type="text/css">\n\t\t@media print {\n\t\t\t@page { margin: 0; size:'
```

Comment:
This is a good place for `.format()` or a [f-strings](https://realpython.com/python-f-strings/)